.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,32 @@
-By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
+**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**
+
+## Description
+
+*Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.*
+
+Fixes # (issue)
+
+
+## New/Edited policies (Delete if not relevant)
+
+### Description
+*Include a description of what makes it a violation and any relevant external links.*
+
+### Fix
+*How do I fix the issue in code and/or in runtime?*
+
+### Category
+Kubernetes, IAM, Networking, Storage, Compute, Public, Secrets, Supply Chain, Monitoring
+
+### Severity
+LOW, MEDIUM, HIGH, CRITICAL
```

Comment:
@tsmithv11 What does this severity relate to? the policy severity? If so - checkov doesn't support that...

Reply:
The docs referenced do, but it's a good point. Removed.